jstz/jstz.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for jsTimezoneDetect
+// Project: https://bitbucket.org/pellepim/jstimezonedetect
+// Definitions by: Olivier Lamothe <https://github.com/olamothe/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface JsTimezoneDetect {
+    determine: ()=> {
+      name: ()=> string
+    }
+}
+
+declare var jstz: JsTimezoneDetect;
+
+declare module "jstz" {
```

Comment:
the module name on npm is "jstimezonedetect". also the file name should be changed to `jstimezonedetect`